src/modules/persistent-cart/persistentCartProductList/persistentCartProductList.md

```diff
@@ -108,9 +212,13 @@
   ]}
   onUpdateQuantity={(item, quantity) => console.log('Update quantity', item, quantity)}
   onUpdateSize={(item, size) => console.log('Update size', item, size)}
-  removeItem={() => console.log('Remove item')}
-  hideCartSidebar={() => {}}
-  segmentCartViewed={() => {}}
+  removeItem={(id) => {
+    console.log('Removing item', id)
```

Comment:
Probably remove

Reply:
Oh I just left this to demonstrate how the callback works, this would only show up when viewing the styleguide and doesn't ship with the mirage package

Reply:
Oh nice! That makes sense then